app/models/menu_items_menu.rb

```diff
@@ -0,0 +1,20 @@
+class MenuItemsMenu < ActiveRecord::Base
```

Comment:
Just created a chore for that => https://www.pivotaltracker.com/story/show/95693298